contracts/sol6/KyberNetwork.sol

```diff
@@ -758,11 +758,13 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
             hint
         );
 
+        require(tradeData.tradeWei <= MAX_QTY, "tradeWei > MAX_QTY");
         if (tradeData.tradeWei == 0) {
             return (0, 0);
         }
 
         // platform fee
+        //*
```

Comment:
max(tradeData.tradeWei) = max(validateTradeCalcDestQtyAndFeeData) = MAX_RATE *MAX_QTY * 10 ** (MAX_DECIMAL - ETH_DECIMAL) / PRECISION * BPS =1e57

Reply:
OK
I only now understand this comment.
it means we have 10k reserves. each has 1 bps split. right?